s31324-Patryk_Latkowski/zadania/zajecia_2.txt

```diff
@@ -0,0 +1,32 @@
+Zadanie 1
+Zalogowałem się do szuflandii, utworzyłem drzewo katalogów poleceniem mkdir.
+Zmieniłem uprawnienia folderu "katalog3" na 777 chmod'em. Za pomocą cd
+przeniosłem się do katalogu innego użytkownika. Sprawdzając mój dostęp
+wpisałem "ls -l" i sprawdziłem, który plik/katalog ma "w" w grupie bądź
+wszystkich i w tych plikach byłem wstanie dokonać modyfikacji pliku/katalogu.
+Utworzyłem plik jajako.txt u s26856, ale plik został bestialsko usunięty.
+Próbując usunąć cudzy plik we własnym katalogu nie byłem wstanie tego zrobić.
+
+Zadanie 2
+Wszedłem do katalogu "katalog1" w "katalog 1". Zabrałem wszystkie uprawnienia
+w katalogu bieżącym używając "chmod 000 .". Wszedłem do katalogu
+"../katalog2". Zabrałem wszystkie uprawnienia w katalogu bieżącym używając
+"chmod 000 ..". Polecenie "cd" mogę użyć jeśli komenda nie będzie mnie
+prowadzić do któryś z tych katalogów. Polecenie "chmod" mogę użyć, gdy mam
+uprawnienia do pliku, aktualnie nie mająć uprawnień do tych katalogów nie
+mogę tego wykonać. Czy możemy bezpośrednio przeskoczyć do katalogu ABC/XYZ,
+jeśli nie mamy uprawnienia wstępu do ABC, ale mamy do XYZ? - tak.
+Czy możemy także wrócić korzystając z polecnia cd - tak, ale tylko do
```

Comment:
jeśli do ABC nie masz ale masz podkatalogu XYZ to wywołując `cd -` powrócisz do XYZ ? hmm... ciekawe.